HISTORY.rst

```diff
@@ -4,9 +4,17 @@ History
 
 0.12.1 (2023-06-01)
 -------------------
+This release is largely a bugfix to better stabilize  performance and enhance the documentation.
 
 * Avoid repeatedly calling `xr.open_dataset` in `OutputReader`'s `hydrograph` and `storage` properties. This seems to cause kernel failures in Jupyter notebooks.
 
+Internal changes
+^^^^^^^^^^^^^^^^
+* Hyperlinks to documented functions now points to entries in the `User API` section.
+* Docstrings are now more conformant to numpy-docstring conventions and formatting errors raised from badly-formatted pydantic-style docstrings have been addressed.
+* In order to prevent timeout and excessive memory usage, Jupyter notebooks have been adjusted to no longer run on ReadTheDocs. All notebooks have been updated to the latest RavenPu and remain tested against RavenPy externally.
```

Comment:
```suggestion
* In order to prevent timeout and excessive memory usage, Jupyter notebooks have been adjusted to no longer run on ReadTheDocs. All notebooks have been updated to the latest RavenPy and remain tested against RavenPy externally.
```